deploy-agent/deployd/agent.py

```diff
@@ -428,6 +492,10 @@ def main():
     else:
         agent.serve_once()
 
+    create_sc_gauge('deployd.stats.internal.time_elapsed_proc_sec',
+                    agent.stat_time_elapsed_internal(),
```

Comment:
instead using the magic __call__ function why not just have a get_duration function or something? I think that will make the code more readable.

Reply:
updated per req